recipes/scikit-build/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "scikit-build" %}
+{% set version = "0.4.0" %}
+{% set sha256 = "88d4a5f8db51bf0031b7f8ed7db902678815a38706055e11e528707b32191df2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/scikit-build/scikit-build/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script:
+    - python setup.py install --single-version-externally-managed --record record.txt
+    # Tests are skipped for now.
+    - py.test -k "not (test_push_dir or test_fortran_compiler)"
```

Comment:
This should be in the test block (not the build block) and should be under the "commands" block (which lives at the same level as "imports")